en/integrations/plugin-packs/procedures/operatingsystems-linux-nrpe3.md

```diff
@@ -0,0 +1,275 @@
+
+| Alias                      | Linked by default |
+| :------------------------- | :---------------: |
+| Cmd-Generic                |                   |
+| Connections-Generic        |                   |
+| Cpu-Detailed               |                   |
+| Cpu                        |         X         |
+| Disk-Generic-Name          |                   |
+| Disk-Global                |                   |
+| Disk-IO-Generic-Name       |                   |
+| Disk-IO-Global             |                   |
+| File-Date-Generic          |                   |
+| File-Size-Generic          |                   |
+| Inodes-Generic-Name        |                   |
+| Inodes-Global              |                   |
+| Is-File-Generic            |                   |
+| Is-Not-File-Generic        |                   |
+| Load                       |         X         |
+| Memory                     |         X         |
+| Packet-Errors-Generic-Name |                   |
+| Packet-Errors-Global       |                   |
+| Process-Generic            |                   |
+| Swap                       |         X         |
+| Traffic-Generic-Name       |                   |
+| Traffic-Global             |                   |
+| Uptime                     |         X         |
+
+### Monitored metrics
+
+Here are the metrics monitored in the "Link by default" services.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Cpu-->
+
+| Metric          | UOM | Description               |
+| :-------------- | :-: | :------------------------ |
+| `cpu0`          |  %  | Average core 0 CPU usage  |
+| `cpu1`          |  %  | Average core 1 CPU usage  |
+| ...             |  %  | ...                       |
+| `total_cpu_avg` |  %  | Average overall CPU usage |
+
+<!--Load-->
+
+| Metric   | UOM | Description                            |
+| :------- | :-: | :------------------------------------- |
+| `load1`  |     | System load average on last 1 minute   |
+| `load5`  |     | System load average on last 5 minutes  |
+| `load15` |     | System load average on last 15 minutes |
+
+<!--Memory-->
+
+| Metric   | UOM | Description                           |
+| :------- | :-: | :------------------------------------ |
+| `buffer` |  B  | Amount of memory allocated to buffers |
+| `cached` |  B  | Amount of memory allocated to cache   |
+| `slab`   |  B  | Amount of memory allocated to Slab    |
+| `used`   |  B  | Total amount of used memory           |
+
+<!--Swap-->
+
+| Metric      | UOM | Description      |
+| :---------- | :-: | :--------------- |
+| `free`      |  B  | Free Swap space  |
+| `used`      |  B  | Used Swap space  |
+| `used_prct` |  %  | Swap space usage |
+
+<!--Uptime-->
+
+| Metric   | UOM | Description                    |
+| :------- | :-: | :----------------------------- |
+| `uptime` |  s  | Elapsed time since last reboot |
```

Comment:
Use the perfdata nlabel instead of legacy perfdata label as often as possible in the first column. 

UOM dedicated column sounds great, we will implement this in the next monitoring proc and update guidelines accordingly. Could you please just make it the last column on the right?